include/ppconsul/consul.h

```diff
@@ -225,12 +225,22 @@ namespace ppconsul {
         template<class... Params, class = kwargs::enable_if_kwargs_t<Params...>>
         std::string makeQuery(const Params&... params) const
         {
-            return parameters::makeQuery(kw::dc = dataCenter(), kw::token = defaultToken(), params...);
+            // Exclude kw::token from query (should be submitted as a header)
+            return parameters::makeQuery(kw::dc = dataCenter(), params..., kw::token = "");
```

Comment:
It will work like this but a better solution would be to introduce a keywords operation to remove a keyword from the parameters pack. I'm fine with the current approach but if you want to do some template magic then you can implement a keyword removal operation :)

Reply:
This is probably not As straightforward as I thought. There is no way to nicely unpack a bunch of values into function arguments except via wrapping the function in lambda. Perhaps the simplest way would be to introduce a keyword tombstone so it’s possible to write something like `kw::token = kwargs::none`.

Anyway, this is not a blocker and should be addressed separately.